diff --git a/Source/Core/Common/Assembler/GekkoIRGen.cpp b/Source/Core/Common/Assembler/GekkoIRGen.cpp
index f7f62d3d7f..eeff502900 100644
--- a/Source/Core/Common/Assembler/GekkoIRGen.cpp
+++ b/Source/Core/Common/Assembler/GekkoIRGen.cpp
@@ -537,8 +537,9 @@ void GekkoIRPlugin::AddSymbolResolve(std::string_view sym, bool absolute)
       sym.size(),
   };
 
+  // MSVC 19.41 requires an explicit parameter list before mutable.
   m_fixup_stack.emplace(
-      [this, sym, absolute, source_address, err_on_fail = std::move(err_on_fail)] mutable {
+      [this, sym, absolute, source_address, err_on_fail = std::move(err_on_fail)]() mutable {
         auto label_it = m_labels.find(sym);
         if (label_it != m_labels.end())
         {
@@ -574,19 +575,19 @@ void GekkoIRPlugin::AddNumLabelSymResolve(std::string_view sym, u32 num)
 
   // Searching forward only
   size_t search_start_idx = static_cast<size_t>(m_numlabs.size());
-  m_fixup_stack.emplace(
-      [this, num, source_address, search_start_idx, err_on_fail = std::move(err_on_fail)] mutable {
-        for (size_t i = search_start_idx; i < m_numlabs.size(); i++)
-        {
-          if (num == m_numlabs[i].first)
-          {
-            return m_numlabs[i].second - source_address;
-          }
-        }
+  m_fixup_stack.emplace([this, num, source_address, search_start_idx,
+                         err_on_fail = std::move(err_on_fail)]() mutable {
+    for (size_t i = search_start_idx; i < m_numlabs.size(); i++)
+    {
+      if (num == m_numlabs[i].first)
+      {
+        return m_numlabs[i].second - source_address;
+      }
+    }
 
-        m_owner->error = std::move(err_on_fail);
-        return u32{0};
-      });
+    m_owner->error = std::move(err_on_fail);
+    return u32{0};
+  });
 }
 
 void GekkoIRPlugin::SaveOperandFixup(size_t str_left, size_t str_right)
